server/service/src/sync/translations/stock_line.rs

```diff
@@ -62,6 +62,7 @@ impl SyncTranslation for StockLineTranslation {
             NameTranslation.table_name(),
             StoreTranslation.table_name(),
             LocationTranslation.table_name(),
+            BarcodeTranslation.table_name(),
```

Comment:
Noticed this while fixing the integration tests, the dependency was missing here...